docs/developer/howtos/README.md

```diff
@@ -0,0 +1,16 @@
+# Developer Howtos
+
+Practical Guides to solving practical problems.
+
+  - [How to work with the acceptance tests](acceptance_tests.md)
+  - [How to make Python-based applications work](custom-python-builder.md)
+  - [How to work with Epinio during Development](development.md)
+  - [How to add a user for API access](new-api-user.md)
+  - [What happens inside when deploying a development binary](behind-the-curtains.md)
+
+[comment1]: # Comment is done by abusing link label syntax (without a link referencing it)
+[comment2]: # .
+[comment3]: # One of the things is not like the others. IOW misclassified, I believe.
+[comment4]: # See also issue https://github.com/epinio/epinio/issues/698#issuecomment-900893447
+[comment5]: # where it is argued that the behind the curtains text should be folded into the
+[comment6]: # development document.
```

Comment:
I don't understand this section. What is this about? Also it doesn't render well (I clicked on the three-dots menu -> view file).

Reply:
It is an inline markdown comment. As a comment it should not render at all.
That said, I better remove it and make a ticket out of it.

Reply:
Ticket is #721 . Removal will be committed when the new probe endpoint is done.